daphne/src/vdaf/mod.rs

```diff
@@ -531,7 +531,13 @@ impl VdafConfig {
         let mut transitions = Vec::with_capacity(num_reports);
         for report_share in agg_job_init_req.report_shares.iter() {
             if processed.contains(&report_share.report_metadata.id) {
-                return Err(DapAbort::UnrecognizedMessage);
+                return Err(DapAbort::UnrecognizedMessage {
+                    detail: format!(
+                        "report ID {} already appeared in the current Aggregate initialization request being processed by the Helper",
```

Comment:
This works, but it can be simplified a bit more :)
```suggestion
                        "report ID {} appears twice in the same aggregation job",
```